examples/signalfx/signalfx-collector.yaml

```diff
@@ -0,0 +1,75 @@
+extensions:
+  health_check:
+  zpages:
+receivers:
+  otlp:
+  sapm:
+  signalfx:
+  # This section is used to collect OpenTelemetry metrics
+  # Even if just a SignalFx µAPM customer, these metrics are included
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'otel-collector'
+          scrape_interval: 10s
+          static_configs:
+            - targets: ['localhost:8888']
+              # If you want to use the environment filter
+              # In the SignalFx dashboard
+              #labels:
+                #environment: demo
+          metric_relabel_configs:
+            - source_labels: [ __name__ ]
+              regex: '.*grpc_io.*'
+              action: drop
+processors:
+  batch:
+  # Optional: If you have a different environment tag name
+  # If this option is enabled it must be added to the pipeline section below
+  #attributes/copyfromexistingkey:
+    #actions:
+    #- key: environment
+      #from_attribute: YOUR_EXISTING_TAG_NAMEE
+      #action: upsert
+  # Optional: If you want to add an environment tag
+  # If this option is enabled it must be added to the pipeline section below
+  #attributes/newenvironment:
+    #actions:
+    #- key: environment
+      #value: "YOUR_ENVIRONMENT_NAME"
+      #action: insert
+  queued_retry:
+  # Enabling the memory_limiter is strongly recommended for every pipeline.
+  # Configuration is based on the amount of memory allocated to the collector.
+  # The configuration below assumes 2GB of memory. In general, the ballast
+  # should be set to 1/3 of the collector's memory, the limit should be 90% of
+  # the collector's memory up to 2GB, and the spike should be 25% of the
+  # collector's memory up to 2GB. In addition, the "--mem-ballast-size-mib" CLI
+  # flag must be set to the same value as the "ballast_size_mib". For more
+  # information, see
+  # https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/memorylimiter/README.md
+  memory_limiter:
+    ballast_size_mib: 683
+    check_interval: 2s
+    limit_mib: 1800
+    spike_limit_mib: 500
+exporters:
+  # Traces
+  sapm:
+    access_token: "YOUR_ACCESS_TOKEN"
+    endpoint: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com/v2/trace"
+  # Metrics
+  signalfx:
+    access_token: "YOUR_ACCESS_TOKEN"
+    realm: "YOUR_SIGNALFX_REALM"
+service:
+  pipelines:
+    traces:
+      receivers: [sapm, otlp]
```

Comment:
should enable zipkin to enable reception from istio mixer adapter.

Reply:
good call, done